### PR TITLE
"My orders" module bugfixes (see description)

### PR DESCRIPTION
**Pull request by enkryptor**
- unified desktop layout
- fix the order status display
- add more order control buttons
- disable order wizard for own services

### CODE REVIEW

**Review by vitslepukhin — APPROVED** [review]

File: src/app/order/components/date-time-step/date-time-step.component.ts [hunk 1]
```diff
@@ -22,18 +23,26 @@ import { switchMap } from 'rxjs/operators';
   ],
 })
 export class DateTimeStepComponent extends StepComponent<DateTimeStepData> implements OnInit {
-  public readonly form = this.fb.group({
-    datetime: [null, Validators.required],
+  public readonly formControl = new FormControl(null, Validators.required);
```
[the remark this answers is not in the document]
Comment (enkryptor) [reply src/app/order/components/date-time-step/date-time-step.component.ts:26]: the `formControl` property should be the FormControl class instance, not AbstractControl

File: src/app/order/components/step-container/step-container.component.ts [hunk 2]
```diff
@@ -35,6 +36,10 @@ export class StepContainerComponent implements OnInit, OnDestroy {
   ) {
   }
 
+  @Input() public set disabled(value: boolean) {
```
[the remark this answers is not in the document]
Comment (enkryptor) [reply src/app/order/components/step-container/step-container.component.ts]: it is 👍

File: src/app/my-orders/components/abstract/order-list-item.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { EventEmitter } from '@angular/core';
+
+export abstract class OrderListItem {
+  public abstract statusChanged: EventEmitter<void>;
+  public pending: boolean = false;
+
+  protected async perform(action: () => Promise<void>) {
```
[the remark this answers is not in the document]
Comment (enkryptor) [reply src/app/my-orders/components/abstract/order-list-item.ts:7]: Ionic's interface returns Promise ¯\_(ツ)_/¯

File: src/app/order/components/date-time-step/date-time-step.component.ts [hunk 4]
```diff
@@ -22,18 +23,26 @@ import { switchMap } from 'rxjs/operators';
...
+  public readonly form = new FormGroup({
+    datetime: this.formControl,
   });
   public displayedCalendars$: Observable<MasterCalendar[]>;
+  public isMyService$: Observable<boolean>;
 
   private readonly currentlyViewedDate = new BehaviorSubject<Date>(new Date());
+  private readonly professional$ = new BehaviorSubject<number>(NaN);
```
[the remark this answers is not in the document]
Comment (enkryptor) [reply src/app/order/components/date-time-step/date-time-step.component.ts:34]: The reasoning is reacting to changes either in masters list or service.

**Pull request merged**